plugins.json

```
@@ -12816,6 +12816,110 @@
+    "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-servicenow-v2.3.4.tar.gz",
+    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-servicenow/releases/tag/v2.3.4",
+    "hosting": "",
+    "author_type": "mattermost",
+    "release_stage": "production",
+    "enterprise": false,
+    "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAmOi5z0ACgkQ0bVLR6XO/sSXRw/7BG6CQ3YQT6MzdD57Y7xRJBwNGguYQ1h+Rj9E2XDoAt5THgggoYctMonSMQLNK+9lDSFeZe8s7Ezxu3bHTv+fNTyYflEQamV63+RnDtQOU4M+5D8tq8Qbhk80lb/ARA1btt12emfJnZaEuKkIjY+61ZlTuLuS2foRWu8gfsMCqR5d7/fWp9d46gGpAMG40lKSGo8RtCxD5v41ePFA/tj68VSRCA8ZAHMLQXExHG2Rdqq6xeJzaw0XvuiPDC26rQOzJstb2IqznxZERxb0GfYGTuv7txuAKxBKvJ+6IG259EaJbRR+GLgRLP+pMt5S0H6Ezlq0Bn4gUvo/fO68NsZr8AzIBUq9452f4mO4jcMzf1cyWmZ5zCZeYWwfiRdYH4MIjwLtAHjIzRsaRfxnMRtfCXsZfUAcDs5qCpEBHvA5XSn+Nz0Eay+vo0QRYVNG5Y2Mz/UoVu1n9ar8UlXvPz+0y0ZE+kJyv7xTOXiVmsMEC+xhRne8DeJNkR+fwwbrFO+qrg5G+KH2S3pNJp6mmQ0FMw3TYmkDtgkkKS6FTDexRNrr9BskjYx8m9D8vpzE1vLRBWA1Em5h6qoV5mdwqNZQxyJ+sY61BeCKJG2tOw4f/YowWNKw6OX734qbFVUAgPPc6A4Mufy+DUQus4LcuZNrEwqh/UNrAViK3Z8lEBmkOPg=",
+    "repo_name": "mattermost-plugin-servicenow",
+    "manifest": {
+      "id": "mattermost-plugin-servicenow",
+      "name": "Servicenow Plugin",
```

0/5, not for this PR

```suggestion
      "name": "ServiceNow Plugin",
```

---

Submitted a PR for this https://github.com/mattermost/mattermost-plugin-servicenow/pull/159

---

Good catch :+1: